### ISSUE TITLE
[Native Promise] - Promise.all then callback does not work

### ISSUE THREAD

**Issue by lovedota** [charleshansen/mock-promises#20]
Hi @charleshansen 

I have a case

```
var a = Promise.all([]).then((values) => {
   console.log('ababa', values)
});
mockPromises.executeForResolvedPromises();
mockPromises.executeForPromise(a);
```

no console.log fired !!!!

**Comment by charleshansen** [comment]
Good catch. It looks like it doesn't work if you give Promise.all an empty array. An easy fix, I just pushed it as 0.6.1

**Comment by charleshansen** [comment]
fixed
